cli/src/merge_tools/builtin.rs

```diff
@@ -341,6 +341,16 @@ async fn make_diff_files(
                 sections.extend(make_diff_sections(&old_contents, &new_contents)?);
             }
 
+            (
+                FileContents::Binary {
+                    hash: left_hash, ..
+                },
+                FileContents::Binary {
+                    hash: right_hash, ..
+                },
```

Comment:
nit: require `hash: Some(..)` for a bit of safety?

The hash field is used only for displaying purpose in other places.